Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -929,46 +926,44 @@
+    <string name="debugging_debuglog_legal_section_body">"After the authenticity of your app has been verified, the error report is sent to the RKI through a secure connection. Error reports will only be used for troubleshooting and error correction within the framework of future app updates. Only technical support employees can access error reports. Error reports contain a variety of status messages and events that are triggered in the app, but do not contain any information that would enable the RKI to determine your identity. Only if you name the error report ID in connection with further messages can a connection be made between the message (and your name contained there, for example) and the information contained in the error report (such as technical messages for calculation as part of exposure logging, information displayed and steps performed in the app, and, where applicable, a retrieved test result and random IDs shared as part of notifying others)."</string>
 
     <!-- XHED: Title for Bugreporting share log screen -->
-    <string name="debugging_debuglog_share_log_title">"Fehlerbericht senden"</string>
+    <string name="debugging_debuglog_share_log_title">"Send Error Report"</string>
     <!-- YTXT: First body section for bugreporting share log screen -->
-    <string name="debugging_debuglog_share_log_section_one">"Bevor Sie den aufgezeichneten Fehlerbericht an den technischen Support des RKI übersenden können, ist Ihr Einverständnis erforderlich."</string>
+    <string name="debugging_debuglog_share_log_section_one">"Your consent is needed before you can send the recorded error report to RKI technical support."</string>
     <!-- YTXT: Second body section for bugreporting share log screen -->
-    <string name="debugging_debuglog_share_log_section_two">"Nach der Übersendung erhalten Sie eine Fehlerbericht-ID. Diese können Sie angeben, um dem technischen Support z.B. weitere Informationen zukommen zu lassen und dabei eine Zuordnung zu Ihrem Fehlerbericht zu ermöglichen. Wenn Sie die Fehlerbericht-ID nicht mitteilen, ist dem RKI eine Zuordnung zu Ihrer Person nicht möglich."</string>
+    <string name="debugging_debuglog_share_log_section_two">"After you send the report, you will receive an error report ID. You can specify this ID to provide technical support with further information, for example, and enable its assignment to your error report. If you do not provide the error report ID, the RKI cannot assign the report to you as an individual."</string>
     <!-- YTXT: Privacy Information section for bugreporting share log screen -->
-    <string name="debugging_debuglog_share_log_privacy_information">"Ausführliche Informationen zu dieser Datenverarbeitung und den Datenschutzrisiken in den USA und anderen Drittländern."</string>
+    <string name="debugging_debuglog_share_log_privacy_information">"Detailed Information about This Data Processing and Data Protection Risks in the U.S. and Other Third Countries"</string>
```

Comment:
Do we need a dot at the end?

Reply:
Thanks. I pinged Albert. Will be fixed with the next translation delivery.